modules/claims_api/app/controllers/claims_api/v1/claims_controller.rb

```diff
@@ -24,9 +24,7 @@ def index
       def show
```

Comment:
If we find in the future that consumers would like this endpoint to show details of what the error is when the claim is errored, we could consider doing something like adding a new serializer that would include the value of the `evss_errors` attribute.

Psuedo-Code Idea
```
if claim.status == 'errored
  render json: claim, serializer: ClaimsApi::AutoEstablishedClaimWithErrorsSerializer
elsif
  ...
end
```

Reply:
I think there's still value in conditionally showing any errors and just making it a `200` instead of a `4xx` status code

Reply:
I don't know that we can do that because then we'll create the situation where a 200 response from this endpoint could have either a body that represents a claim, or a body that represents a set of errors.  Which I would think could definitely cause breaking changes for existing consumers.